ldb/models.py

```diff
@@ -250,17 +250,48 @@ class Meta:
 
     @property
     def current_member(self):
-        return (self.date_from is not None and self.date_to is None) or \
-               self.merit_date_from is not None or self.honorary_date_from is not None
+        # Note that the ExportResource class implements the same logic using queries (Q objects)
+        return ((self.date_from is not None and (self.date_to is None or self.date_to > date.today())) or
+                self.merit_date_from is not None or self.honorary_date_from is not None) and not self.person.deceased
```

Comment:
Als de database consistent is met de validatieregels hieronder wel ja, maar dat betwijfel ik..